basics/getting-started/frequent-questions/ingestion-faq.md

```diff
@@ -26,7 +26,39 @@ Pinot automatically detects new partitions in Kafka topics. It checks for new pa
 
 You can configure the interval for this job using the`controller.realtime.segment.validation.frequencyPeriod` property in the controller configuration.
 
-### How do I enable partitioning in Pinot, when using Kafka stream?
+### Does Pinot support multi-column partitioning?
+
+Pinot supports multi-column partitioning for offline tables. Map multiple columns under [`tableIndexConfig.segmentPartitionConfig.columnPartitionMap`](/configuration-reference/table.md#segmentPartitionConfig). Pinot assigns the input data to each partition according to the partition configuration individually for each column.
```

Comment:
The partition column used in assignment is configured under `InstanceReplicaGroupPartitionConfig` which allows only a single partition column. This is because we cannot identify which partition ID to use for the segment when assigning it

Reply:
is this additional information or correcting what's above?

Reply:
Currently Pinot only supports `partition pruning on multiple columns`, and I feel the above doc might cause confusion because we can only use a single partition column to assign a segment. It might be more clear if the title is `Does Pinot support partition pruning on multiple partition columns`

Reply:
thank you! updated